README.md

```diff
@@ -0,0 +1,76 @@
+# Atom Renderer
+
+A library that renders atoms from the content API into front-end code. It is written in
+javascript using type-checking from FlowJS, and wrapped in a Scala app. 
+
+## To Install 
+
+1. Clone repo: `git clone git@github.com:guardian/atom-renderer.git`
+
+2. Install node dependencies: `yarn`*
+
+3. Install sbt dependencies `sbt compile`
+
+## Run FlowJS
+
+Installing the node modules above should have installed what you need to run flow. 
+
+To check for any compile errors, run flow like this: `yarn run flow`
+
+More information on [Getting Started with Flow](https://flow.org/en/docs/getting-started/). 
+
+## Build the app 
+
+1. Build the JS code: `yarn build`
+
+2. Build the Scala code: `sbt run` 
+
+## Preview your changes
+
+In future we may add a dummy atom so the results of your change can be previewed from within the repo. 
+
+For now, you need to publish your changes to NPM as a new version of the Atom Renderer package. Then, you need to open
```

Comment:
Please specify that the semver rules for test packages should be followed here.

Reply:
Scratch that, just saw you did it a few lines below

Reply:
ah sure ;)